### ISSUE TITLE
Can `package.nls.*.json` in a sub directory?

### ISSUE THREAD

**Issue by shalldie** [microsoft/vscode#169150]
Can `package.nls.*.json` in a sub directory?

I wan to put them in a sub directory, there's to be too many files at root.

**Comment by TylerLeonhardt** [comment]
Today, the `package.nls.*.json` files can't be put anywhere else... however, I would like to leverage the `l10n` property of the `package.json` (which is used for the `l10n` VS Code API) to enable folks to put their `package.nls.*.json` files there.

When doing that, maybe we also rename it to `package.l10n.json` and `package.l10n.*.json` so that everything is consistent (of course we would have backwards compatibility)

**Comment by shalldie** [comment]
Sounds great, I like it !
Is it an idea or a work in progress?

**Comment by TylerLeonhardt** [comment]
It's just an idea for now. I've got some other work that is higher up on my list but we'll leave this issue open as a reminder of this work

**Comment by anzory** [comment]
Hi all.
Is it true that if VSCode does not have any localization (e.g. Albanian), it means that it will not appear in env.language, and that it is useless to create bundle.l10n.sq.json bundle - it will not be accepted by l10n anyway?

**Comment by TylerLeonhardt** [comment]
@anzory yes that is correct. An Albanian language pack would need to be created to set VS Code to Albanian and then extensions would be able to support this language.

**Comment by TylerLeonhardt** [comment]
Closing in favor of an issue that has a bit more information.
